data/strings/strings.txt

```diff
@@ -18708,7 +18708,8 @@
 
   [button_layer_subway]
     tags = android,ios
-    en = Subway
+    en = Metro
+    en-US = Subway
```

Comment:
Looks like en-US is not properly supported at the moment by our twine tool. You can see it in its XML output. 
Maybe it would be enough to create a corresponding directory (values-en-rUS), or maybe the tool should be updated.

Reply:
This is a more global issue actually. At the moment, on both iOS and Android, `en` language (`values` dir on Android) is treated as `en-US`. So you can only try to add non-US variants.
Ideally, all translations should work properly: generic en, specified/overrided en-US or en-GB, etc.

Reply:
Hmm, I'll see if I can add en- variations. I agree there should be a generic EN as default

Reply:
Yes, twine should be checked, but it may already work.
A proper fix requires:
1. Making locale-aware localizations for the US.
2. Testing them on both platforms (to enable a newly added folder on iOS, it should be selected/added also in XCode project).

An imperfect workaround is to leave en as is (the US), and translate/add GB and other localizations. It would be great to follow a perfect way.